Documentation/ignition.md

```diff
@@ -1,16 +1,16 @@
-# Ignition
+# Container Linux Configs
 
-Ignition is a system for declaratively provisioning disks during the initramfs, before systemd starts. It runs only on the first boot and handles partitioning disks, formatting partitions, writing files (regular files, systemd units, networkd units, etc.), and configuring users. See the Ignition [docs](https://coreos.com/ignition/docs/latest/) for details.
+A Container Linux Config is a YAML document which declares how Container Linux instances' disks should be provisioned on network boot and first-boot from disk. Users can declare disk paritions, write files (regular files, systemd units, networkd units, etc.), and configure users. See the Container Linux Config [spec](https://coreos.com/os/docs/latest/configuration.html).
 
-## Fuze configs
+### Ignition
 
-Ignition 2.0.0+ configs are versioned, *machine-friendly* JSON documents (which contain encoded file contents). Operators should write and maintain configs in a *human-friendly* format, such as CoreOS [fuze](https://github.com/coreos/fuze) configs. As of `matchbox` v0.4.0, Fuze configs are the primary way to use CoreOS Ignition.
+Container Linux Configs are validated and converted to *machine-friendly* Ignition config (JSON) by matchbox when serving booting machines. This means, matchbox users need only write Container Linux Configs. The Ignition executor in the Container Linux initamfs implements declarative configuration.
```

Comment:
Agreed, how do we usually talk about Ignition, the binary shipping with Container Linux?